### PR TITLE
Added summary and modals taglib + new js components improvements

### PR DESCRIPTION
**Pull request by fabio-d-m**

### CODE REVIEW

**Comment by marco-leo** [comment]
ci:test

File: commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/model/SummaryItem.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ */
+
+package com.liferay.commerce.frontend.model;
+
+import com.liferay.portal.kernel.util.Validator;
+
+/**
+ * @author Fabio Diego Mastrorilli
+ */
+public class SummaryItem {
```
Comment (marco-leo) [inline commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/model/SummaryItem.java:22]: Summary is a very generic name, can we be more specific?

File: commerce-frontend-js/src/main/resources/META-INF/resources/components/modal/Modal.es.js [hunk 2]
```diff
@@ -65,13 +71,13 @@ const Modal = props => {
...
-				break;
+		
+		if(iframeLoadingCounter > 1) {
+			_handleFormSubmit();
+		}
+		
+		if(iframeLoadingCounter === 1) {
+			_handleIframeFirstLoad();
 		}
 		// eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (marco-leo) [inline commerce-frontend-js/src/main/resources/META-INF/resources/components/modal/Modal.es.js:82]: do we need this comment?

File: commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/model/SummaryItem.java (hunk 1 above)
Comment (fabio-d-m) [reply commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/model/SummaryItem.java:22]: Yes why not, ideas? I reflected on java the name I used for the FE component :/

File: commerce-frontend-js/src/main/resources/META-INF/resources/components/modal/Modal.es.js (hunk 2 above)
Comment (fabio-d-m) [reply commerce-frontend-js/src/main/resources/META-INF/resources/components/modal/Modal.es.js:82]: Not mandatory but it hides a eslint warning. I would keep it there

**Comment by marco-leo** [comment]
send to adminui branch

**Pull request closed without merging**